### ISSUE TITLE
[JB] Run Shortcuts in the background

### ISSUE THREAD

**Issue by zanodor** [macstories/obsidian-shortcut-launcher#9]
I listened to (some of) the podcast where Federico was talking about the CLI mode of running shortcuts on MacOS.  
It could be possible on iOS running the Springcuts tweak on jailbroken devices.  
Unfortunately I lack the expertise to whomp together anything of the sort and I doubt it if you were interested to make anything like this publicly available, but at least you could give some pointers how it could be done.  
Basically, to my understanding, the applet should be invoked through terminal/zsh (in `usr/bin`) and shortcuts could be run directly. 

Any thoughts on this?

Cheers

**Comment by andrewcrook** [comment]
> CLI mode of running shortcuts on MacOS

The Shortcuts CLI program on macOS is separate from the Shortcuts App itself. The cli is under `/usr/bin/shortcuts` not ` /Applications/Shortcuts\ Backup.app/`. It's compiled for a different archeture so will not run under  iOS. I doubt that the shortcuts app on iOS is also compiled with any support for the CLI. Not only that, but iOS also has a lot of differences to macOS on how it handles applications. Sorry I don't thinks its possible.
